workshop-prerequisites.html

```diff
@@ -0,0 +1,251 @@
+      </a>
+      <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+      <span class="sr-only">Toggle navigation</span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
+      </button>
+      </div>
+
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav">
+          <li><a href="index.html#journey">Journey</a></li>
+          <li><a href="index.html#speakers">Keynotes</a></li>
+          <li><a href="index.html#schedule">Schedule</a></li>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">Events<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="https://in.pycon.org/cfp/devsprint-2018/proposals/">Devsprints</a></li>
+              <li><a href="open-spaces.html">Open Spaces</a></li>
+            </ul>
+          </li>
+          <li><a href="index.html#sponsors">Sponsors</a></li>
+          <li><a href="https://in.pycon.org/blog/">Blog</a></li>
+          <li class="dropdown">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">Attend<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="index.html#tickets">Tickets</a></li>
+              <li><a href="index.html#venue">Venue</a></li>
+              <li><a href="travel.html">Traveling to Hyderabad</a></li>
+              <li><a href="hotels.html">Hotels</a></li>
+              <li><a href="https://goo.gl/forms/iqZ8I3xfzcXXLxQq2">Childcare</a></li>
+            </ul>
+          </li>
+          <li><a href="faq.html">FAQ</a></li>
+          <li class="dropdown active">
+            <a class="dropdown-toggle" data-toggle="dropdown" href="#">More<span class="caret"></span>
+            <ul class="dropdown-menu">
+              <li><a href="volunteer.html">Volunteer</a></li>
+              <li><a href="https://goo.gl/forms/R4g5sxjZIXezvp7z1">Community Spaces</a></li>
+              <li><a href="coc.html">COC</a></li>
+              <li><a href="index.html#contact">Contact</a></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <!-- Section: about -->
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 class="small-caps center">Workshop Prerequisites</h2>
+        <p class="mb-4">
+          Below are the prerequisites detailed for the workshops either by a Keynote or a Sponsor. All other workshop details can be seen from the <a href="https://in.pycon.org/cfp/2018/proposals/"><strong>CFP</strong></a> itself.
+        </p>
+        <hr>
+        <h3>Jupyter hub and Binder by Carol Wiling</h3>
```

Comment:
Can we add a `<section>` tag with a id here ? So that we can directly link to sections using `workshop-prerequisites.html#section1` Do the same for all the sections.

Reply:
@ananyo2012 problem with on this page is we're not getting to the exact content and it's been cropped because of the navbar. I tried it earlier as well.

Reply:
Can you increase the padding a bit and try if it looks good ?